ClassMoodApp/static/js/controllers.js

```diff
@@ -73,9 +74,9 @@ app.controller('dialogController', function($scope, $log, $http) {
         var className = $scope.classname;
         var studentID = 1;
         if (className !== '') {
-            $http.post('/set_student_lectures/' + className + '/' + studentID)
+            $http.post('/set_student_classes/' + className + '/' + studentID)
             .then(function successfulCallback(response) {
-                if (response.data === true) {
+                if (response.data == 'true') {
```

Comment:
Unfortunately this is correct. Angular documentation shows that the response data is held as a string (https://docs.angularjs.org/api/ng/service/$http) and logging the typeof response.data confirms this. We can either do a string comparison here or use the transform method to convert the string response from the server to a boolean.